### PR TITLE
flameshot: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Another page seems to have escaped into here.

File: pages/linux/flameshot.md [hunk 1]
```diff
@@ -4,30 +4,34 @@
...
+`flameshot screen --number {{5}}`
 
-- Set the save path to write screenshots to:
+- Create a screenshot and print it to the standard output:
 
-`flameshot full --path {{path/to/directory}}`
+`flameshot gui --raw`
 
-- Delay the screenshot for N milliseconds and output to clipboard:
+- Create a screenshot and copy it to a clipboard:
```
Comment (mfrw) [inline pages/linux/flameshot.md:31]:
```suggestion
- Create a screenshot and copy it to the clipboard:
```

Comment (EmilySeville7cfg) [reply pages/linux/flameshot.md:31]: I think it's better to use `a` here: we are not talking about a specific clipboard.

Comment (EmilySeville7cfg) [reply pages/linux/flameshot.md]: I thought there are [several][wiki] types of clipboard in linux. 🤔 

[wiki]: https://en.wikipedia.org/wiki/Clipboard_(computing)#:~:text=The%20X%20Window%20System%20commonly,various%20selections%20is%20not%20standardized.

**Review by mfrw — APPROVED** [review]
LGTM

File: pages/linux/flameshot.md [hunk 2]
```diff
@@ -4,30 +4,34 @@
...
+
+- Create a screenshot interactively in a simplified mode:
 
-- Take a screenshot from monitor 1:
+`flameshot launcher`
+
+- Create a screenshot from a specific monitor:
 
-`flameshot screen --number {{1}}`
+`flameshot screen --number {{5}}`
```
Comment (navarroaxel) [inline pages/linux/flameshot.md:25]: How many screens do you have? :laughing: 
```suggestion
`flameshot screen --number {{2}}`
```

Comment (EmilySeville7cfg) [reply pages/linux/flameshot.md:25]: I have one screen. 😆

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**